.env

```
@@ -1,6 +1,8 @@
 DB_PASSWORD=D1ymf8wyQEGthFR1E9xhCq
+
 DB_USER=LTIdbUser
 DB_NAME=LTIdb
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_USER key

(UnorderedKey)